### PR TITLE
chore: 🤖 add semantic release bot with docker extension

### PR DESCRIPTION
**Pull request by polymath-eric**
### JIRA Link 

[DA-438](https://polymesh.atlassian.net/browse/DA-438)

### Changelog / Description 

automatically publish docker images

### Checklist - 

- [ ] New Feature ?
- [ ] Updated swagger annotation (if API structure is changed) ?
- [ ] Unit Test (if possible) ?
- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

**Comment by polymath-eric** [comment]
The security hotspot from sonar is false positive BTW. It thinks I am hard coding a password, but I am configuring the tool to look for the env to substitute in.

I think someone else should mark it as false positive since its my PR and seems like dismissing a security concern is a bad habit.

**Review by VictorVicente — APPROVED** [review]

File: release.config.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  branches: [
+    'master',
+    {
+      name: 'beta',
+      prerelease: true,
+    },
+    {
+      name: 'alpha',
+      prerelease: true,
+    },
```
Comment (prashantasdeveloper) [inline release.config.js:12]:
```suggestion
    {
      name: 'develop',
      prerelease: true,
    },
```

File: release.config.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  ],
+  /*
+   * In this order the **prepare** step of @eclass/semantic-release-docker, will run first
+   * followed by @semantic-release/github:
+   *  - Update the package.json version and create the docker image
+   *  - Push a release commit and tag, including configurable files
+   *
+   * See:
+   *  - https://github.com/semantic-release/semantic-release/blob/beta/docs/usage/plugins.md#plugin-ordering
+   *  - https://github.com/semantic-release/semantic-release/blob/beta/docs/extending/plugins-list.md
```
Comment (prashantasdeveloper) [inline release.config.js:22]: These URL don't exist anymore

**Review by prashantasdeveloper — CHANGES_REQUESTED** [review]

File: release.config.js (hunk 1 above)
Comment (polymath-eric) [reply release.config.js:12]: I think we should switch over to the alpha / beta / master model we have in other repositories so its consistent. I made new branches in this repo and was going to tell people to switch when this got merged.

I am not expecting to use them much, but they could be nice when supporting stuff across chain versions.

Comment (polymath-eric) [reply release.config.js:12]: Will take out beta

**Review by prashantasdeveloper — APPROVED** [review]
LGTM

**Comment by polymath-eric** [comment]
The real test will be on merge.

The bot refused to go all the way since it was in a PR even when I added the branch name into the releases as a test.

**Pull request merged**